fenix/app/src/main/java/org/mozilla/fenix/browser/BaseBrowserFragment.kt

```diff
@@ -429,6 +429,7 @@ abstract class BaseBrowserFragment :
 
         _browserToolbarView = BrowserToolbarView(
             context = context,
+            navController = findNavController(),
```

Comment:
Let's not pass in the `navController` to the view. We can use the existing `BrowserToolbarInteractor` which has access to navController via `BrowserToolbarController`